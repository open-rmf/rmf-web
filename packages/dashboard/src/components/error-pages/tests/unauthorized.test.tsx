import React from 'react';
import { render } from '@testing-library/react';
import Unauthorized from '../unauthorized';

describe('Unauthorized', () => {
  test('renders correctly', () => {
    const root = render(<Unauthorized />);
    expect(root.queryByText('Unauthorized')).toBeTruthy();
    root.unmount();
  });
});
