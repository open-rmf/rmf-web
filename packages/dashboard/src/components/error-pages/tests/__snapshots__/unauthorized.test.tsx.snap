// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Unauthorized renders correctly 1`] = `"<span>Unauthorized</span>"`;
